### DISCUSSION TITLE
Is there an asset re-confirmation feature?

### DISCUSSION THREAD

**Discussion by airsky98** [snipe/snipe-it#15875] [General]
The use case is let's say the team (or individual) requested for a physical server or virtual machines for testing. Eventually, it might turn into production or the project might be abandoned after some testing or even the person in charge might leave the company. 

Potentially, the asset becomes a stray asset without any ownership. Only after some time, when we need the resource then we will ask around whether this asset is still in use. If not, we will reclaim it. 

It will be great that we can get the owners (a team or individual) to acknowledge it every 3 months for example. Making sure that the resource/asset is still in use.  

I'm aware there's an audit feature but does it make the user acknowledge themselves that the asset is still in use?

**Comment by gebauer** [comment]
Hi,

I am looking for something similar. I manage keys with this software, and like to ask the users once in a year to confirm that they still own the keys...


Best
Jan
